src/modules/twap/pure/PrimaryActionButton/getTwapFormState.tsx

```diff
@@ -3,28 +3,35 @@ import { TWAPOrder } from '../../types'
 
 export interface TwapFormStateParams {
   isSafeApp: boolean
+  isFallbackHandlerSetupAccepted: boolean
   verification: ExtensibleFallbackVerification | null
   twapOrder: TWAPOrder | null
 }
 
 export enum TwapFormState {
-  LOADING_SAFE_INFO,
-  NOT_SAFE,
-  ORDER_NOT_SPECIFIED,
-  NEED_FALLBACK_HANDLER,
-  CAN_CREATE_ORDER,
+  LOADING_SAFE_INFO = 'LOADING_SAFE_INFO',
+  NOT_SAFE = 'NOT_SAFE',
+  NEED_FALLBACK_HANDLER = 'NEED_FALLBACK_HANDLER',
+  ACCEPTED_FALLBACK_HANDLER_SETUP = 'ACCEPTED_FALLBACK_HANDLER_SETUP',
```

Comment:
Any specific reason to switch the enum from number to strings?

Reply:
Yes :) 
```

export enum TwapFormState {
  LOADING_SAFE_INFO,
  NOT_SAFE,
  NEED_FALLBACK_HANDLER,
  ACCEPTED_FALLBACK_HANDLER_SETUP,
}

const state = TwapFormState. LOADING_SAFE_INFO

if (!state) {
console.log('Falsy falsy check, because LOADING_SAFE_INFO === 0')
}
```